lib/internal/per_context/primordials.js

```diff
@@ -63,10 +95,18 @@ function copyPropsRenamedBound(src, dest, prefix) {
     if ('get' in desc) {
       copyAccessor(dest, prefix, newKey, desc);
     } else {
-      if (typeof desc.value === 'function') {
-        desc.value = desc.value.bind(src);
+      const { value } = desc;
+      if (typeof value === 'function') {
+        desc.value = value.bind(src);
+      }
+
+      const name = `${prefix}${newKey}`;
+      ReflectDefineProperty(dest, name, desc);
+      if (varargsMethods.includes(name)) {
+        ReflectDefineProperty(dest, `${name}Apply`, {
+          value: applyBind(value, src),
```

Comment:
```suggestion
          value: applyBind(value),
```

Reply:
Same here, except this one is also specified as binding the `this`.